.gitignore

```diff
@@ -30,3 +30,5 @@ settings.yaml
 gdrive_credentials
 client_secrets.json
 
+.vscode/settings.json
```

Comment:
Could we please just exclude the whole subtree, I have no feeling if there will be some more files generated under certain conditions